playground/main.ts

```diff
@@ -150,4 +151,4 @@ if (typeof URL !== 'undefined') {
     render = false;
   }
 }
-render && renderPlaygroundWidget();
+render && renderPlaygroundWidget(window.additionalOptions ?? {});
```

Comment:
@aarongranick-okta @lesterchoi-okta I'm trying to add a way to override / add to widget options from our e2e testcafe tests, so I'm relying on an optional global here that we set in the `clientScripts` in the test itself (see below). Let me know if you disagree on this approach or if there's a better way.